store/tikv/coprocessor.go

```diff
@@ -525,7 +518,14 @@ func (it *copIterator) Next(ctx context.Context) ([]byte, error) {
 func (it *copIterator) handleTask(bo *Backoffer, task *copTask, ch chan copResponse) {
 	remainTasks := []*copTask{task}
 	for len(remainTasks) > 0 {
+		startTime := time.Now()
 		tasks, err := it.handleTaskOnce(bo, remainTasks[0], ch)
+		costTime := time.Since(startTime)
```

Comment:
We should move the metrics into handleTaskOnce(). Because the it may be blocked [here](https://github.com/pingcap/tidb/blob/master/store/tikv/coprocessor.go#L591).

Reply:
@shenli 
There is no channel.

Reply:
The channel is in the handleCopResponse(). Please see: https://github.com/pingcap/tidb/blob/master/store/tikv/coprocessor.go#L665